lib/bundler/definition.rb

```diff
@@ -829,7 +829,8 @@ def expand_dependencies(dependencies, remote = false)
             "The dependency #{dep} will be unused by any of the platforms Bundler is installing for. " \
             "Bundler is installing for #{@platforms.join ", "} but the dependency " \
             "is only for #{dep.platforms.map {|p| Dependency::PLATFORM_MAP[p] }.join ", "}. " \
-            "To add those platforms to the bundle, run `bundle lock --add-platform #{dep.platforms.join ", "}`."
+            "To add those platforms to the bundle, " \
+            "run `bundle lock --add-platform #{dep.platforms.map {|p| Dependency::PLATFORM_MAP[p] }.join " "}`."
```

Comment:
Maybe extract the array from map into a variable since it's also used on the line above?